src/MFRC522Extended.cpp

```diff
@@ -828,19 +824,19 @@ MFRC522::StatusCode MFRC522Extended::TCL_Transceive(TagInfo *tag, byte *sendData
 		*backLen = in.inf.size;
 		memcpy(backData, in.inf.data, in.inf.size);
 	}
-
+	
 	// Check chaining
 	if ((in.prologue.pcb & 0x10) == 0x00)
 		return result;
-
 	// Result is chained
 	// Send an ACK to receive more data
 	// TODO: Should be checked I've never needed to send an ACK
+
 	while (in.prologue.pcb & 0x10) {
```

Comment:
In `TCL_TransceiveRBlock` the var `linked` is set:
```c++
*linked = in.prologue.pcb & 0x10;
```
and then at the end of the while it is checked
```c++
if(!linked) break; 
```
But that's the same as
```c++
while (in.prologue.pcb & 0x10)
```

Or did I missed something?

Reply:
Ah, I see. You are right.